ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-eus-4.12.yaml

```diff
@@ -187,14 +195,33 @@ tests:
     - ref: nutanix-upgrade-pre-config
     - chain: openshift-upgrade-qe-test-eus
     workflow: cucushift-installer-rehearse-nutanix-ipi-proxy
-- as: vsphere-ipi-f28
-  cron: 28 6 18 * *
+- as: vsphere-agent-sno-f14
```

Comment:
`sno` cluster could not do paused pool update.

Reply:
oh, yes, make sense